catalog/ui/catalog-ui-search/src/main/webapp/component/query-interactions/query-interactions.hbs

```diff
@@ -39,20 +39,24 @@
         Duplicate
     </div>
 </div>
-<div class="query-interaction interaction-deleted" title="Search Archived" data-help="Executes the search, but specifically looks for archived results.">
-    <div class="interaction-icon fa fa-recycle">
-    </div>
-    <div class="interaction-text">
-        Search Archived
-    </div>
-</div>
-<div class="query-interaction interaction-historic" title="Search Historical" data-help="Executes the search, but specifically looks for historical data.">
-    <div class="interaction-icon fa fa-history">
-    </div>
-    <div class="interaction-text">
-        Search Historical
-    </div>
-</div>
+{{#if isVersioningEnabled}}
```

Comment:
❓ So if `isVersioningEnabled` is `false`, then search archived and historical are both unavailable. But why aren't we checking to see that `isSearchArchivedEnabled` before showing the search archive option like we do for search historical?

Reply:
this has been changed to use css classes, but the logic is that versioning must be enabled by having a Historian for Search Archived or Search Historical to even be considered.